rows.go

```diff
@@ -28,13 +28,6 @@ var (
 	maxChunkDownloaderErrorCounter = 5
 )
 
-// SnowflakeRows provides an API for methods exposed to the clients
-type SnowflakeRows interface {
```

Comment:
Instead of deleting this API, it would be a smaller diversion to rename our SnowflakeRows export to something else.

Reply:
OK I un-deleted this interface and renamed our exported type.